### PR TITLE
add progress bar, opacity, gradient

### PR DESCRIPTION
**Pull request by ajlstep**

### CODE REVIEW

**Comment by ajlstep** [comment]
commect 1

**Comment by ajlhimik** [comment]
ddsa

File: lib/const/const.dart [hunk 1]
```diff
@@ -87,3 +87,16 @@ class CColors {
...
   // static const largeTitleColor = Color(0xFFC4C4C4);
 }
+
+class CGradients {
+  static const Gradient whiteImageGradient = LinearGradient(
+    colors: [
+      Color.fromRGBO(37, 40, 73, 1),
+      Color.fromRGBO(59, 62, 91, 0.08),
+    ],
+    begin: Alignment.topCenter,
```
Comment (ajlhimik) [inline lib/const/const.dart:97]: ppdpsa sad

File: lib/widgets/card_widget.dart [hunk 2]
```diff
@@ -10,95 +10,114 @@ class SightCard extends StatelessWidget {
...
-              Padding(
-                padding: const EdgeInsets.fromLTRB(16, 16, 16, 0),
-                child: Row(
-                  crossAxisAlignment: CrossAxisAlignment.start,
-                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                  children: [
-                    Text(
-                      sight.type.name.toString(),
-                      style: CTextStyles.smallBold.copyWith(
-                        color: CColors.white,
```
Comment (ajlhimik) [inline lib/widgets/card_widget.dart:37]: comment

**Pull request merged**